tests/integration/targets/object_storage_info/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+vultr_object_storage_info_stores:
+  - label: '{{ vultr_resource_prefix }}-object-storage-info1'
```

Comment:
minor: would you mind using `"` instead of `'` (seems yaml linter action is missnig, it is part of the integration tests but of course, these don't run on external PRs)

Reply:
Fixed.